Source/SwiftLintFramework/Rules/RuleConfigurations/TrailingWhitespaceConfiguration.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TrailingWhitespaceConfiguration.swift
+//  SwiftLint
+//
+//  Created by Reimar Twelker on 12.04.16.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+
+public struct TrailingWhitespaceConfiguration: RuleConfiguration, Equatable {
+    var severityConfiguration = SeverityConfiguration(.Warning)
+    var ignoresEmptyLines = false
+
+    public var consoleDescription: String {
+        return "ignores empty lines: \(ignoresEmptyLines ? "true" : "false")"
```

Comment:
`consoleDescription` is used on output of `swiftlint rules`. So, `ignores_empty_lines` would be better that same as keyword used in configuration file.